### ISSUE TITLE
Use K2 compiler, improves build speed upto 2x

### ISSUE THREAD

**Issue by MV-GH** [LemmyNet/jerboa#1172]
### Pre-Flight checklist

- [X] Did you check to see if this issue already exists?
- [X] This is a single feature request. (Do not put multiple feature requests in one issue)
- [X] This is not a question or discussion. (Use https://lemmy.ml/c/jerboa for that)

### Describe The Feature Request Below

https://android-developers.googleblog.com/2023/07/try-k2-compiler-in-your-android-projects.html

**Comment by dessalines** [comment]
They say it shouldn't be used for production builds yet. Build time isn't too much an issue atm.

**Comment by MV-GH** [comment]
Yeah when its production ready we will use it. For just JVM it won't have much impact. But for my kmp lib it has significant improvement l.

**Comment by MV-GH** [comment]
This is done now.
